edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/Program.cs

```diff
@@ -24,7 +25,7 @@ public class Program
 
         public static int Main()
         {
-            Console.WriteLine($"[{DateTime.UtcNow.ToString("MM/dd/yyyy hh:mm:ss.fff tt")}] Edge Agent Main()");
+            Console.WriteLine($"{DateTime.UtcNow.ToString("yyyy-MM-dd HH:mm:ss.fff zzz", CultureInfo.InvariantCulture)} Edge Agent Main()");
```

Comment:
Nice to have: suggest to make time format as a constant to share across multiple projects.

Reply:
Yeah, would have been nice but it is a little tricky to use in the logger template itself. 
So added an method on DateTime itself that can be reused